### ISSUE TITLE
Process change: Champions and assigned maintainers.

### ISSUE THREAD

**Issue by eeue56** [elm-community/Manifesto#16]
So, here's some process change I would like to happen to elm-community:

Right now, there's a lot of PRs and issues with no-one merging or doing anything about it. The problem is ownership. Each repo is not "owned" by a single person, so there is not a single person to act as the representive for that repo. That needs to change. So, instead, this should happen:
- If a repo is proposed to be moved to elm-community, we must assign a "champion" to that repo. The champion will have full support to merge PRs as they seem fit.
- Other community members can ping the champion if they want something to get merged. 
- If there's something that really needs to get merged, and the champion has taken more than X days to respond, we can merge things anyway.
- Unresponsive champions will be emailed. Lack of response will mean a new champion will be assigned to that repo.

This based loosely off how package maintance works in Fedora/RH/Debian.

tl;dr - we need assigned maintainers who can be decisive and responsive.

@elm-community/packages 
@elm-community/manifesto 
@elm-community/guides

**Comment by eeue56** [comment]
I'll give this another day to sit here. The next actions:
- Dole out maintainer responsibilities
- Create a document for all our repos, naming a maintainer for each. It'll look like `repo | maintainer github name | maintainer email address`
- Tada

**Comment by rgrempel** [comment]
I think this is all a very good idea.

**Comment by eeue56** [comment]
Everyone,

I've added a PR [here](https://github.com/elm-community/Manifesto/pull/17) where I have documented the new responsibilities of a maintainer. I also think we should get rid of the current groups (as they aren't needed in the new world). I've also added a tl;dr.

Finally, I've added a document with the current repos owned by elm-community. Every one is unmaintained until someone makes a PR to change that ;)

**Comment by jeremy-w** [comment]
How will discoverability of this maintainer system work if you're a person looking at one of the maintained repos? Do we plan to have each repo's README clearly call out the maintainer and link to their responsibilities at the top?

My concern is that the central list is good for administering this but not so good for discoverability if you're a non-maintainer community member.

**Comment by eeue56** [comment]
If you are interested enough to become a maintainer, then you will look on this repo.

**Comment by rgrempel** [comment]
I wonder whether @jeremy-w's suggestion might be useful?

At least, it would seem sensible, in a project's README, to indicate who the maintainer is -- or, if there is none, that the project could use one.

Not that we necessarily need to go through each README and change them right now. But it might be a useful idiom to adopt over time.

**Comment by rgrempel** [comment]
Thinking this over a little more, there was one additional comment I wanted to make.

In some cases, I think the ideal development process has been where two people keep an eye on a repository, and they merge each other's PR's (and those coming from other people as well, of course). Or, at least comment on each other's PR's.

Now, identifying a primary maintainer has obvious benefits, and I'm not arguing against it. I suppose that all I'm saying is that we shouldn't discourage people from adopting a "don't merge your own PR" philosophy if they can find a partner to work with on something.

**Comment by eeue56** [comment]
Considering this issue closed
